geonode/resource/manager.py

```diff
@@ -55,7 +58,9 @@
 from ..base import enumerations
 from ..base.models import ResourceBase
 from ..layers.metadata import parse_metadata
+from ..layers.models import Layer
 
+from geonode.geoserver.helpers import gs_catalog
```

Comment:
This is breaking the "resource_manager" contract.
We don't wont GeoServer specific logic on the generic RM interface.